### DISCUSSION TITLE
RedirectUL

### DISCUSSION THREAD

**Discussion by anvog777** [Web3Auth/Web3Auth#642] [Help]
Hi there,

I am implementing web3auth on React Native 0.7.1 (without expo).

The login works fine but gets stuck on the "Redirect to app" button.

The demo version that I downloaded from GitHub works fine.

I followed the documentation https://web3auth.io/docs/sdk/react-native/initialize and the Integration Builder.

I configured the redirectURL on AndroidManifest.xml:

`<data android:scheme="w3a"/>`

And on my login.js file:

```
const scheme = 'w3a'; // Or your desired app redirection scheme
const resolvedRedirectUrl = `${scheme}://openlogin`;
```

Am I missing something?

Thanks!

**Comment by shahbaz17** [comment]
Hey @anvog777 

Please update to the latest version of [React Native SDK](https://github.com/Web3Auth/web3auth-react-native-sdk) `v3.1.0` and reach out to us if this error is still there.
